### PR TITLE
Add testing in travis with different versions

### PR DESCRIPTION
**Pull request by federicoaldunate**
Add testing in travis with different versions of `rails`, `activeadmin` and `ruby`
- Add `database_cleaner` is necessary for 5.0.x rails version
- `rails` versions are `5.0.x`, `5.1.x` and `5.2.x`
- `activeadmin` versions are `1.0.0`, `1.2.1` and `1.4.3`
- `ruby` versions are `2.2.10`, `2.3.8`, `2.4.3`, `2.5.3` and `ruby-head`

### CODE REVIEW

File: spec/dummy/config/secrets.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+development:
+  secret_key_base: b9ed8530bded47950895f26188cc21a984e271e262d8e8b8c78c8fc57c65a65fd5818db617906414b9f8b5f608bdc115af3aaa1714431e66aa538c2b8c749304
```
Comment (santib) [inline spec/dummy/config/secrets.yml]: Why do you need this?

File: spec/features/chat_page_spec.rb [hunk 2]
```diff
@@ -22,7 +22,6 @@
 
     expect(page).to have_content(person1.email)
     expect(page).to have_content(person2.email)
-
```
Comment (santib) [inline spec/features/chat_page_spec.rb]: why? I think it's clearer with the empty line. Same below in the PR

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -48,6 +48,26 @@
...
+    DatabaseCleaner.clean_with(
+      :truncation,
+      except: %w[ar_internal_metadata]
+    )
+  end
+
+  config.before :each do
+    DatabaseCleaner.strategy = :truncation
+    DatabaseCleaner.start
+    ActionMailer::Base.deliveries.clear
```
Comment (santib) [inline spec/spec_helper.rb]: not needed, right?

File: spec/dummy/config/secrets.yml (hunk 1 above)
Comment (federicoaldunate) [reply spec/dummy/config/secrets.yml]: It's needed for versions 5.0 and 5.1 of rails

**Pull request merged**